orchagent/acltable.h

```diff
@@ -27,6 +29,78 @@ typedef enum
     ACL_STAGE_EGRESS
 } acl_stage_type_t;
 
+typedef enum
+{
+    ACL_TABLE_UNKNOWN,
+    ACL_TABLE_L3,
+    ACL_TABLE_L3V6,
+    ACL_TABLE_MIRROR,
+    ACL_TABLE_MIRRORV6,
+    ACL_TABLE_MIRROR_DSCP,
+    ACL_TABLE_PFCWD,
+    ACL_TABLE_CTRLPLANE,
+    ACL_TABLE_DTEL_FLOW_WATCHLIST,
+    ACL_TABLE_DTEL_DROP_WATCHLIST
+} acl_table_type_t;
+
 typedef map<string, acl_stage_type_t> acl_stage_type_lookup_t;
 
-#endif /* SWSS_ACLTABLE_H */
+class AclOrch;
```

Comment:
I think it is better to include 'aclorch.h' in this file as that would a superset for other things as well.

Reply:
I think it's generally not a good practice to mutually include two header files.

Reply:
We have portsorch.h included in most header files. But aside from that, aclorch.h has most of the references for acltable.h.. My point is here as per description, the aim is to reduce the file size and not separate out the functionality.

Reply:
sorry i didn't quite get it. i think there's no duplication between the acltable.h and aclorch.h files.

Reply:
I agree Shuotian, I don't think having aclorch include acltable and vice-versa is a good idea. 

From what I can tell the only dependency that AclTable has on AclOrch is `m_mirrorTableCapabilities[ACL_TABLE_MIRRORV6]` and `m_isCombinedMirrorV6Table` in create(). I think it would be cleaner to pass those two variables into create() (or the constructor if you think they might be useful in other methods as well) and then remove the reference to m_pAclOrch entirely. That way there's no cyclical dependency between AclTable and AclOrch.